tests/integration-tests/clusters_factory.py

```diff
@@ -243,11 +243,12 @@ def __init__(self, keep_logs_on_failure=False):
         self.__created_clusters = {}
         self._keep_logs_on_failure = keep_logs_on_failure
 
-    def create_cluster(self, cluster, extra_args=None):
+    def create_cluster(self, cluster, extra_args=None, suppress_exception=False):
```

Comment:
Minor: we usually name `suppress_exception` as `raise_on_error` in our code repo. Would be nice if we can try to keep consistency here, but not required.

Reply:
Done